crux-metrics/src/crux/metrics.clj

```diff
@@ -18,28 +20,57 @@
                :deps #{:crux.node/node :crux.node/indexer :crux.node/bus :crux.node/kv-store}}})
 
 (def jmx-reporter
-  {::jmx-reporter {:start-fn (fn [{::keys [registry]} {::keys [jmx-reporter-opts]}]
-                               (doto (jmx/reporter registry
-                                                   (merge {:domain "crux.metrics"}
-                                                          jmx-reporter-opts))
-                                 jmx/start))
+  {::jmx-reporter {:start-fn (fn [{::keys [registry]} args]
+                               (jmx/start-reporter registry args))
+                   :args {:crux.metrics.dropwizard.jmx/domain {:doc "Add custom domain"
+                                                               :required? false
+                                                               :crux.config/type :crux.config/string-vector}
+                          :crux.metrics.dropwizard.jmx/rate-unit {:doc "Set rate unit"
+                                                                  :required? false
+                                                                  :crux.config/type :crux.config/string}
```

Comment:
I don't know what I'd pass for these units, as a user?

Reply:
ah, `TimeUnit`, seen it below. in that case, might be worth us creating another type for them?

at the very least, we should give examples, or point the user to the TimeUnit enum, in the `:doc`